sandbox/extract-imagery-metadata_stevendepaschalis.R

```diff
@@ -0,0 +1,104 @@
+# This script has two purposes: 1. Demonstrate how to run functions to extract metadata from imagery
+# dataset EXIF data, and 2. Demonstrate how to write a new metadata extraction function.
+
+# --- Setup ---
+
+# Load all the functions (and package dependencies) of this R package
+devtools::load_all()
+
+# Define the root of the local data directory
+datadir = readLines(file.path("sandbox", "data-dirs", "steven-metadata-laptop.txt"))
+
+exif_file = exif_files[1]
+exif = prep_exif(exif_file)
+
+# Load necessary packages
+library(sf)
+
+# Define the function to extract metadata including resolution, aspect ratio, image file format, and mean image frequency
+extract_metadata = function(exif) {
```

Comment:
Please name this function `extract_metadata_sd` so that everyone working on this project can have their own functions and not conflict by creating multiple functions with the same mane. Then we can merge them later. Also note that as requested in the general comments, the content of this function should be converted into a few separate sub-functions, and this function `extract_metadata_sd` should simply be a wrapper around them. For example, see `extract_metadata_dy` in R/imagery-metadata-extraction_dy.R.